packages/features/bookings/Booker/components/DatePicker.tsx

```diff
@@ -0,0 +1,37 @@
+import { shallow } from "zustand/shallow";
```

Comment:
Tiny wrapper around existing datepicker component. This components provides all data necessary for the booker, this way colocates everything involved in the booker datepicker into one isolated file for readability.